elide-spring/elide-spring-boot-autoconfigure/src/main/java/com/yahoo/elide/spring/config/ElideAutoConfiguration.java

```diff
@@ -56,6 +58,10 @@
      * @return An instance of ElideDynamicEntityCompiler.
      * @throws Exception Exception thrown.
      */
+
+     private final AbstractJpaTransaction.JpaTransactionCancel jTC = (e) -> { e.unwrap(Session.class).cancelQuery(); };
```

Comment:
Because the signatures are the same, you may be able to define this once as a Function rather than the concrete types above - and just use the single reference below.